venues/MIDL.io/2019/Conference/python/create-review-invitations.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+
+###############################################################################
+# ex. python create-invitations.py --conf MyConf.org/2017 --baseurl http://localhost:3000
+#       --username admin --password admin_pw
+#
+# To be run after submission due date to create review invitations for all the papers.
+# For each paper:
+# 1) create authorGroup (can see reviews, can't write a review)
+#           reviewer group (reviewers for this paper)
+#           and nonReviewerGroup (folks that aren't allowed to read the review at least not yet)
+# 2) create review invitation
+###############################################################################
+
+## Import statements
+import argparse
+import sys
+import openreview
+from openreview import tools
+import config
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print("Connecting to "+client.baseurl)
+
+builder = openreview.conference.ConferenceBuilder(client)
+builder.set_conference_id(config.CONFERENCE_ID)
```

Comment:
I guess you need to set the headers too otherwise the home page is overwritten with the default data.

Reply:
Then we should move the header info into the config.py file so we don't have to update it in a bunch of places.

Reply:
That is a good idea, may be the config file should have a function to get the conference instance.